src/applications/edu-benefits/tests/feedback-tool/00.feedback-tool.cypress.spec.js

```diff
@@ -0,0 +1,152 @@
+import mockFeedbackPost from './fixtures/mocks/feedback-post.json';
+import mockFeedbackGet from './fixtures/mocks/feedback-1234.json';
+import Timeouts from 'platform/testing/e2e/timeouts';
+import testData from './schema/maximal-test.json';
+
+describe('Feedback Tool Test', () => {
+  it('Fills the form and navigates accordingly', () => {
+    cy.intercept('POST', '/v0/gi_bill_feedbacks', { body: mockFeedbackPost });
+    cy.intercept('GET', '/v0/gi_bill_feedbacks/1234', mockFeedbackGet);
+
+    cy.visit('/education/submit-school-feedback');
+    cy.get('body').should('be.visible', { timeout: Timeouts.normal });
```

Comment:
`{ timeout: Timeouts.normal } `should be the default, no?

Reply:
I'm more than happy to reevaluate this as it's something that was carried over from being in place on Nightwatch (and exists in many other Cypress tests as well). The Cypress default per their docs is `4000` and our default `normal` is `2000` per our helpers file, so we're applying a shorter timeout than the default in this case.  If we feel the default timeout will cover all these cases I'm happy to remove it wherever it's unnecessary.

Reply:
Personally I'd rather just use the Cypress default. If we want to redefine the default for all tests, we should do that in our Cypress config, rather than adding an override to every `get`.

Reply:
I agree, we should use the Cypress defaults unless there's a particular reason not to. The timeouts from Nightwatch aren't going to be as relevant with the removal of Nightwatch itself, so I would recommend redefining any nececssary custom timeouts around the Cypress defaults instead.

Reply:
Thanks for the feedback from you both.  I went ahead and removed any instances of `Timeout: normal` from Cypress get functions across the code base.  I'll run it in CI a few times to be sure that nothing has been unintentionally impacted here.  As many other files were affected, this will require a few other team reviews to go through, but we'll have much cleaner code on the other side of it!

Reply:
Done (details in other thread)